src/applications/vaos/new-appointment/components/DateTimeRequestPage/index.jsx

```diff
@@ -87,9 +86,12 @@ export function DateTimeRequestPage({
         showValidation={submitted && !userSelectedSlot(selectedDates)}
       />
       <FormButtons
-        onBack={() => routeToPreviousAppointmentPage(history, pageKey)}
+        onBack={() =>
+          dispatch(routeToPreviousAppointmentPage(history, pageKey))
+        }
         onSubmit={() =>
           goForward({
+            dispatch,
             data,
             history,
             routeToNextAppointmentPage,
```

Comment:
I think you can remove this line, since we're using the `routeToNextAppointmentPage` that's imported at the top of the file, rather than pulling this from the parameters for `goForward`

Reply:
Done